guide/14-deep-learning/inference-only-text-models-in-arcgis-learn.ipynb

```diff
@@ -0,0 +1,987 @@
+{
```

Comment:
<p>The <strong>TextSummarizer</strong> model can be used generate <s>summaries</s> summary for a given <s>input</s> text. The model has been fine-tuned on <strong>summarization</strong> task. Visit <a href="https://huggingface.co/models?filter=summarization" target="_blank">this</a> link to learn more about the available models for <strong>summarization</strong> task. To get a list of supported transformer backbones for this model use the below command.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Made the necessary changes</p>